test/example.test.js

```diff
@@ -1,18 +1,36 @@
 // IMPORT MODULES under test here:
 // import { example } from '../example.js';
+import { getRandomThrow } from '../get-random-throw.js';
+import { checkResults } from '../check-results.js';
```

Comment:
great job refactoring and getting these tests done! You both accomplished so much during that lab time